engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/telemetry/TelemetryLicenseKeyDto.ftl

```diff
@@ -0,0 +1,39 @@
+<#macro dto_macro docsUrl="">
+<@lib.dto>
+
+    <@lib.property
+        name = "customer"
+        type = "string"
+        desc = "The name of the customer the license was issued for."/>
+
+    <@lib.property
+        name = "type"
+        type = "string"
+        desc = "The license type."/>
+
+        <@lib.property
+        name = "valid-until"
+        type = "string"
+        format = "date-time"
```

Comment:
What does this format mean? Is the usage correct here?

Reply:
You are right, it should be `date` as defined in the [OpenAPI spec](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#data-types). This way OpenAPI knows the format will be `YYYY-MM-DD`.